src/pools/D3MAavePool.sol

```diff
@@ -66,6 +66,47 @@ interface LendingPoolLike {
     );
 }
 
+// Need to use a struct as too many variables to return on the stack
+struct ReserveDataV3 {
+    //stores the reserve configuration
+    uint256 configuration;
+    //the liquidity index. Expressed in ray
+    uint128 liquidityIndex;
+    //the current supply rate. Expressed in ray
+    uint128 currentLiquidityRate;
+    //variable borrow index. Expressed in ray
+    uint128 variableBorrowIndex;
+    //the current variable borrow rate. Expressed in ray
+    uint128 currentVariableBorrowRate;
+    //the current stable borrow rate. Expressed in ray
+    uint128 currentStableBorrowRate;
+    //timestamp of last update
+    uint40 lastUpdateTimestamp;
+    //the id of the reserve. Represents the position in the list of the active reserves
+    uint16 id;
+    //aToken address
+    address aTokenAddress;
+    //stableDebtToken address
+    address stableDebtTokenAddress;
+    //variableDebtToken address
+    address variableDebtTokenAddress;
+    //address of the interest rate strategy
+    address interestRateStrategyAddress;
+    //the current treasury balance, scaled
+    uint128 accruedToTreasury;
+    //the outstanding unbacked aTokens minted through the bridging feature
+    uint128 unbacked;
+    //the outstanding debt borrowed against this asset in isolation mode
+    uint128 isolationModeTotalDebt;
+}
+
+// Aave Lending Pool v3
+// Interface changed slightly from v2 to v3
+interface LendingPoolReserveDataV3Like {
+    function POOL_REVISION() external view returns (uint256);
```

Comment:
This seems unused.

Reply:
[done](https://github.com/makerdao/dss-direct-deposit/pull/93/commits/17c7a15b8706ed516f58f7de63e64e5a1d3701f8)